modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/DumpCore_conf.pm

```diff
@@ -538,7 +538,7 @@ sub pipeline_analyses {
         {
             -logic_name        => 'report_failed_assembly_chain',
             -module            => 'Bio::EnsEMBL::DataCheck::Pipeline::EmailNotify',
-            -parameters        => { 'email' => $self->o('email') },
+            -parameters        => { 'email' => $self->o('email').",".$self->o('web_email') },
```

Comment:
If the value is empty, therefore that will make `xxx@xxx.com,` Not sure its has an impact.